src/hooks/engine.rs

```diff
@@ -321,6 +321,34 @@ pub static paintbuffer: Pointer<*mut [portable_samplepair_t; 1026]> =
     Pointer::empty(b"paintbuffer\0");
 pub static paintedtime: Pointer<*mut c_int> = Pointer::empty(b"paintedtime\0");
 pub static realtime: Pointer<*mut f64> = Pointer::empty(b"realtime\0");
+
+pub static R_DrawSequentialPoly: Pointer<
+    unsafe extern "C" fn(*mut c_void, *mut c_int) -> *mut c_void,
+> = Pointer::empty_patterns(
+    b"R_DrawSequentialPoly\0",
+    // Stolen from BunnymodXT
+    Patterns(&[
+        // 6153
+        pattern!(55 8B EC 51 A1 ?? ?? ?? ?? 53 56 57 83 B8 F8 02 00 00 01 75 63 E8 ?? ?? ?? ?? 68 03 03 00 00 68 02 03 00 00),
+        // 4554
+        pattern!(A1 ?? ?? ?? ?? 53 55 56 8B 88),
+        // OpposingForce
+        pattern!(A1 ?? ?? ?? ?? 53 55 BD 01 00 00 00 8B 88 F8 02 00 00 56 3B CD 57 75 62 E8 ?? ?? ?? ?? 68 03 03 00 00 68 02 03 00 00),
```

Comment:
```suggestion
```
This pattern doesn't match outside of WON clients and most likely will never be used